webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -9,10 +9,18 @@
     @vdropzone-thumbnail="transformImage"
   >
     <div class="crop-overlay" ref="cropperOverlay" v-show="showCropper">
-      <ds-button @click.stop.prevent="cropImage" class="crop-confirm" primary>
+      <base-button @click="cropImage" class="crop-confirm" filled>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:48:21Z" title="Tuesday, January 14th 2020, 6:48:21 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -9,10 +9,18 @@
     @vdropzone-thumbnail="transformImage"
   >
     <div class="crop-overlay" ref="cropperOverlay" v-show="showCropper">
-      <ds-button @click.stop.prevent="cropImage" class="crop-confirm" primary>
+      <base-button @click="cropImage" class="crop-confirm" filled>
```


I'm curious how you are getting away with removing the `.stop.prevent` from this...
we added it because we noticed that if you edit a post, and add a different image, and confirm crop, it would submit the form, but I've double checked and it doesn't happen anymore

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:54:09Z" title="Wednesday, January 15th 2020, 8:54:09 am +01:00">Jan 15, 2020</time>_

---


It's complicated and I don't fully understand it myself – it has to do with the implementation of `ds-button` vs `base-button`. The `ds-button` was wrapped in a `div` which handled the `click` event whereas the `base-button` has a native html `button` element at its root and can therefore use the native `click` event.

I tried to find more info on these handlers (`stop`, `prevent`, `capture` etc.) but the docs aren't very helpful, so I tried a few things and ended up with the above solution. If any of the buttons stop working after merging this PR it's one of the first things I would check, though. 😅